modules/core/shared/src/main/scala/scaladex/core/api/PaginationSchema.scala

```diff
@@ -0,0 +1,16 @@
+package scaladex.core.api
+
+import endpoints4s.algebra.JsonEntitiesFromSchemas
+import scaladex.core.model.search.Pagination
+
+trait PaginationSchema extends JsonEntitiesFromSchemas {
```

Comment:
Putting pagination schema into a trait so it can be mixed into other APIs that may use it in the future.